scripts/remove/authelia.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+#
+# |& was added to Bash 4 as an abbreviation for 2>&1 |. - https://tldp.org/LDP/abs/html/io-redirection.html
```

Comment:
mmmmmm are we sure everyone is on bash4 actually? something something debian

Reply:
something something google it.

https://packages.debian.org/jessie/bash